CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+## [4.1.1] -
```

Comment:
Not sure about the next release date

Reply:
That's OK, we usually leave it blank or set it to `TBD` until just before we're ready to release and know the date (or we do a small separate PR which updates the version and date in thie file).

Reply:
This is probably a `4.2.0` release since (if I understand it correctly) it's a backwards-compatible new feature and not a bug fix.

Reply:
will do